optimum/intel/ipex/inference.py

```diff
@@ -87,7 +90,19 @@ def __enter__(self):
                             )
 
                             # Enable automatic mixed precision (AMP) if we are going to target `bfloat16`
-                            with torch.cpu.amp.autocast(enabled=(self._dtype == torch.bfloat16)):
+                            with torch.cpu.amp.autocast(enabled=(self._dtype == torch.bfloat16)), torch.no_grad():
+                                if self._model.tokenizer is not None:
+                                    try:
+                                        jit_inputs = []
+                                        dummy_input = self._model.tokenizer("")
+                                        for key in dummy_input:
+                                            jit_inputs.append(torch.ones((1, len(dummy_input[key])), dtype=torch.long))

```

Comment:
Dummy inputs can also be generated using the optimum.exporters appropriated [configuration](https://github.com/huggingface/optimum/blob/v1.6.1/optimum/exporters/onnx/__main__.py#L77-#L78) 
```
dummy_inputs = config.generate_dummy_inputs(framework="pt")
```
It will be extended in the future but would would require us to add `onnx` as a new dependecy until then